security/nbde_tang_server_operator/nbde-tang-server-operator-release-notes.adoc

```diff
@@ -0,0 +1,44 @@
+//NBDE Tang Server Operator Release Notes
+:_mod-docs-content-type: ASSEMBLY
+[id="nbde-tang-server-operator-release-notes"]
+= NBDE Tang Server Operator release notes
+:context: nbde-tang-server-operator-release-notes
+include::_attributes/common-attributes.adoc[]
+
+toc::[]
+
+The following release notes track the development of the Security Profiles Operator in the OpenShift Container Platform.
+
+* link:https://access.redhat.com/errata/RHEA-2023:120658-03[RHEA-2023:120658-03 - Release of the NBDE Tang Server Operator 1.0.0]
+
+// the customer-facing link must be updated
+
+
+////
+
+[id="nbde-tang-server-operator-release-notes-1-0-0"]
+== NBDE Tang Server Operator 1.0.0
+
+The following advisory is available for the NBDE Tang Server Operator 1.0.0:
+
+* link:https://access.redhat.com/errata/RHBA-2023:XXXX[RHBA-2023:XXXX NBDE Tang Server Operator Bug Fix Update]
```

Comment:
Should the first letters of "Bug Fix Update" be lowercase?

Reply:
@nalhadef I was trying to follow what I've seen in the already existing sections, for example [1]. I'll change it to the sentence case if this is a convention for the release-notes sections.

[1] https://access.redhat.com/documentation/en-us/openshift_container_platform/4.14/html/security_and_compliance/file-integrity-operator#file-integrity-operator-release-notes

Reply:
I don't know about standards for release notes, but sentence case is standard for regular doc headings. I always assumed that link text was in the same class.

Reply:
It seems that this is not consistent across openshift-docs. In [1], sentence case prevails. So I will change it as you are suggesting.

[1] https://access.redhat.com/documentation/en-us/openshift_container_platform/4.14/html/security_and_compliance/compliance-operator#compliance-operator-release-notes

Reply:
👍